invenio_communities/config.py

```diff
@@ -35,9 +35,12 @@
 COMMUNITIES_DELETE_HOLDOUT_TIME = timedelta(days=365)
 """Time after which the communities marked for deletion are hard-deleted."""
 
-COMMUNITIES_LOGO_EXTENSIONS = ['png', 'jpg', 'jpeg']
+COMMUNITIES_LOGO_EXTENSIONS = ['png', 'jpg', 'jpeg', 'svg']
 """Allowed file extensions for the communities logo."""
 
+COMMUNITIES_LOGO_MAX_SIZE = 1000 * 1000 * 1.5  # 1.5 MB
```

Comment:
> which is less "common" unit ...

That might be true, but it would be nice if defaults were in the units used in code (https://github.com/inveniosoftware/invenio-communities/pull/72#discussion_r77604622).

It might be more confusing for a user to see 1.5 MB but not being able to upload it:

```console
$ du -h output.dat
1.5M    output.dat
```